### PR TITLE
added HapticFeedback

### LINKED ISSUE #3026: HapticFeedback
Adding documentation for new functionality:

1. https://github.com/xamarin/Essentials/pull/1138
2. https://github.com/xamarin/Essentials/pull/1333

### PR DESCRIPTION
**Pull request by dimonovdd**
Added documentation for Haptic feedback
Fixes #3026

### CODE REVIEW

**Comment by davidbritch** [comment]
@Redth Any comments on this?

**Comment by Redth** [comment]
LGTM! 🚢

**Comment by PRMerger19** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/essentials/haptic-feedback.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 title: "Xamarin.Essentials: HapticFeedback"
-description: "TThis document describes the HapticFeedback class in Xamarin.Essentials, which lets you perform the haptic feedback."
+description: "This document describes the HapticFeedback class in Xamarin.Essentials, which lets you perform haptic feedback."
 ms.assetid: 7E8B24C4-2625-4DAE-A129-383542D34F1E
-author: dimonovdd
```
Comment (dimonovdd) [inline docs/essentials/haptic-feedback.md]: @davidbritch 
Excuse me, but why was my name changed to another which is not me?

Comment (davidbritch) [reply docs/essentials/haptic-feedback.md]: @dimonovdd Apologies. I'm so used to ensuring that author and ms-author are identical that I edited both of them. I'll put you back as author (ms-author has to be a Microsoft employee though).

Comment (dimonovdd) [reply docs/essentials/haptic-feedback.md]: Thanks

**Comment by davidbritch** [comment]
This will get merged once the haptic feedback functionality is in a Xamarin.Essentials release.

**Comment by jamesmontemagno** [comment]
this can be pulled in. i have a few changes, but will make in another PR as I can't make edits to this one.

**Review by davidbritch — APPROVED** [review]

**Pull request merged**